lib/scripts/controllers/options.js

```diff
@@ -285,6 +293,18 @@ export class OptionCtrl {
     }
   }
 
+  updatePlaybackOptions(){
+    var playbackOpts = JSON.stringify({
+      'volume': this.volume,
+      'rate': this.rate,
+      'voiceName': this.voiceName,
+      'pitch': this.pitch
+    });
+    this.save({
```

Comment:
It does show the message a lot of times, but other event directives don't work well with the range HTML5 input. For ng-blur you have to click outside for it to lose focus, in case of ng-click if you drag the cursor out of the slider while dragging it doesn't update the value etc 

But I don't think it should be a problem because I don't see the user changing the pitch, rate and volume a lot of times.